include/openthread/platform/radio.h

```diff
@@ -68,11 +68,12 @@ extern "C" {
 enum
 {
     OT_RADIO_FRAME_MAX_SIZE     = 127,                            ///< aMaxPHYPacketSize (IEEE 802.15.4-2006)
+    OT_RADIO_CHANNEL_PAGE       = 0,                              ///< 2.4 GHz IEEE 802.15.4-2006
```

Comment:
Shall we define a platform API to get the supported channel page instead of define this as constant?